index.js

```diff
@@ -62,7 +68,8 @@ module.exports = app => {
           tag_name: releaseInfo.tag,
           body: releaseInfo.body,
           draft: true,
-          prerelease: config.prerelease
+          prerelease: config.prerelease,
+          ...(targetBranch ? { target_commitish: targetBranch } : {})

```

Comment:
To improve readability, depends whether the rest api accepts empty string. Though that is easily testable.
```suggestion
          target_commitish: config['target-branch']
```

Reply:
Since the behaviour of passing `null` or `''` to the REST service is undocumented in https://developer.github.com/v3/repos/releases/#create-a-release (or even https://octokit.github.io/rest.js/#octokit-routes-repos-create-release), I don't think it is wise to expect it to always fallback to the default value when such values are passed.

Reply:
If `config['target-branch']` instead would default to `context.payload.repository.default_branch` then it would follow the rest api standard.